examples/test_numba_extension.py

```diff
@@ -0,0 +1,16 @@
+import numba
+from clifford._numba_multivector import MultiVectorType
+
+from numba.extending import lower_getattr
+from numba import types
+from clifford import MultiVector
+import numpy as np
+
+@numba.njit
+def add(a):
+    return MultiVector(a.layout, a.value)
```

Comment:
Something screwy is going on here - if I change this to `a.value * 2`, I get memory filled with `0xDE` . I think the array is being freed or something?